lib/ring_logger/client.ex

```diff
@@ -149,6 +149,15 @@ defmodule RingLogger.Client do
     GenServer.call(client_pid, {:save, path})
   end
 
+  @spec grep_metadata(GenServer.server(), atom(), any(), RingLogger.client_options()) :: [
+          binary()
+        ]
+  def grep_metadata(client_pid, key, match_value, opts) do
+    {_io, to_print} = GenServer.call(client_pid, {:grep_metadata, key, match_value, opts})
+
+    to_print
```

Comment:
```suggestion
    {io, to_print} = GenServer.call(client_pid, {:grep_metadata, key, match_value, opts})

    pager = Keyword.get(opts, :pager, &IO.binwrite/2)
    pager.(io, to_print)
```

We should probably match the pager support here we do with other functions

Reply:
Sounds good